integration-tests/src/test/resources/test.properties

```diff
@@ -19,6 +19,9 @@ tt.failed.tests.if.throwable.classes=java.sql.SQLRecoverableException
 tt.failed.tests.if.throwable.messages=RetryUnderTest,test_FailedToPassedHistoryWithRetry
 tt.failed.tests.max.retries=1
 
+# Class exeptions for source code snippet in report
+tt.report.source.class.exclusions=AbstractTest
```

Comment:
I would prefer having a regular expression for the whole package path.
So you could write `tt.report.source.exclusion.package.regex=\.(?:utils|tools)\.`